client/lib/setup/index.coffee

```diff
@@ -49,6 +54,10 @@ module.exports = (app) ->
   app.use bucketAssets()
   setupAuth app
   app.use locals
+
+  if SENTRY_PRIVATE_DSN
+    app.use RavenServer.errorHandler()
```

Comment:
Just an FYI, Sentry error-handling must come before all other error handling on the server: https://docs.sentry.io/clients/node/integrations/express/